CHANGES.rst

```diff
@@ -4,7 +4,11 @@ Changelog
 5.0.16 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Notify PropertiesUpdated event when member properties are changed
+  [ezvirtual]
+
+- Add simple test for PropertiesUpdated event
```

Comment:
I would not add a changelog for adding a test. It is fine to do this when adding a test is the only thing that you do in a PR. But in this PR, adding a test is just a normal part of the change that you propose.

Reply:
Also very helpful feedback, I'll keep that in mind :)